### DISCUSSION TITLE
Cross-Origin Request Blocked

### DISCUSSION THREAD

**Discussion by talhabw** [supabase/supabase#3077] [Questions] [answered]
Hi. I am using Vue 3 and Supabase-js to access my supabase database. It works fine on my machine but when I upload it to Vercel, I am getting this error:

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at [my supabase database url]`

How can I fix this?

**Comment by silentworks** [comment, marked as answer]
You probably need to add the Vercel domain to the Additional Redirect URLs in your Supabase > Authentication > Settings.

**Reply by AkaCoder404** [reply]
How is done on self-hosted run by `npx supabase init` and `npx supabase start`?
